spicy/runtime/src/driver.cc

```diff
@@ -21,6 +21,14 @@ using hilti::rt::fmt;
 
 using namespace spicy::rt;
 
+#ifndef NDEBUG
+#define DRIVER_DEBUG(...) debug(__VA_ARGS__)
```

Comment:
Won't this break if we ever have to call e.g., `this->debug(...)`? Maybe we could make the forwarding in our `debug` (the non-`virtual` one) conditional on `NDEBUG` so it becomes a no-op for release code.

Reply:
It has to be done before we call any debug(), and the virtual is also called directly.

If anything, I'd do two macros, but it's also just an implementation detail of this file at this point, not sure it's worth much more. If it breaks, we'll notice.